tf-modules/monitoring/variables.tf

```
@@ -0,0 +1,8 @@
+
+variable "prefix" {
+  type = string
+}
+
+variable "elasticsearch_alarms" {
+  type = list
```

This type constraint should be tightened:
```suggestion
  type = list(object({ name = string, arn = string }))
```